server/controllers/request.js

```diff
@@ -0,0 +1,67 @@
+const Request = require("../models/Request");
+const Profile = require("../models/Profile");
+const asyncHandler = require("express-async-handler");
+const User = require("../models/User");
+
+// @route POST /requests
+// @desc Create new request
+// @access Public
+exports.createRequest = asyncHandler(async (req, res, next) => {
+  const { requester, sitter, startDate, endDate } = req.body
+  const request = await Request.create({requester, sitter, startDate, endDate});
+
+  if (request) {
+    res.status(200).json({
+      success: {
+        request: {
+          ...request
+        }
+      }
+    })
+  }
+});
+
+// @route PUT /requests/edit/:requestId
+// @desc edit request
+// @access Public
+exports.editRequest = asyncHandler(async (req, res, next) => {
+  const request = await Request.findById(req.params.requestId)
+
+  if (!request) {
+    res.status(404);
+    throw new Error("Request doesn't exist");
+  }
+  const profile = await Profile.findOne({'userId': req.user.id})
+  if (profile.isSitter) {
```

Comment:
The current design will allow any sitter to update other sitters' requests - I think we've addressed this in the controllers PR but I'm not sure why you reverted the validation here :)

Eg: `(request.sitter == profile._id)`